### PR TITLE
Update to 7.13.0

### PR DESCRIPTION
**Pull request by RhCheGithubBot**
Tracking changes for fixing compatibility with upstream 7.13.0-SNAPSHOT. This PR was created automatically by Jenkins from job devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io

### CODE REVIEW

**Comment by Katka92** [comment]
Last run failed because of `Failed to obtain token for ****! Probably user password is incorrect. Continue with other users. ` Running again.

**Comment by Katka92** [comment]
[test]

**Comment by ibuziuk** [comment]
@Katka92 looks like functional tests need to be updated:
```
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ functional-tests ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 5 source files to /root/payload/functional-tests/target/classes
warning: Supported source version 'RELEASE_6' from annotation processor 'org.jvnet.hudson.annotation_indexer.AnnotationProcessorImpl' less than -source '1.8'
/root/payload/functional-tests/src/main/java/com/redhat/che/selenium/core/RhCheLoginPage.java:28: error: [InjectOnMemberAndConstructor] Members shouldn't be annotated with @Inject if constructor is already annotated @Inject
  @Inject private SeleniumWebDriverHelper seleniumWebDriverHelper;
                                          ^
    (see https://errorprone.info/bugpattern/InjectOnMemberAndConstructor)
  Did you mean 'private SeleniumWebDriverHelper seleniumWebDriverHelper;'?
1 error
```

**Comment by Katka92** [comment]
[test]

**Comment by ibuziuk** [comment]
@Katka92 could you please resolve the conflict ^

**Comment by Katka92** [comment]
@ibuziuk The last failure is weird, the rh-che seems to be running, but is not accessible - returning 404. Should I re-run the deployment&tests or will you take a look?

Screenshot from 2020-05-13 10-44-11

**Comment by ibuziuk** [comment]
@Katka92 this is likely related to the UD changes in the upstream which should be addressed by  https://github.com/redhat-developer/rh-che/pull/1890#issue-416783325

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

File: README.md [hunk 1]
```diff
@@ -1,100 +0,0 @@
-[![Master Build Status](https://ci.centos.org/buildStatus/icon?subject=master&job=devtools-rh-che-build-che-credentials-master/)](https://ci.centos.org/job/devtools-rh-che-build-che-credentials-master/)
```
Comment (ibuziuk) [inline README.md:1]: @Katka92 removal of the README does not look expected

Comment (Katka92) [reply README.md]: Thank you for your catch! I've excluded that change from my commit. Sorry for that.

**Comment by ibuziuk** [comment]
[test]

**Comment by Katka92** [comment]
@ibuziuk I reverted the removal of rh-che specific JavaMaven test. I thought it was merged for 7.13.0 but as far as I see it is not included - my fault. The fix should be in 7.14.x so we can remove the rh-che specific JavaMaven test in that version. Sorry for confusion

**Comment by Katka92** [comment]
https://rhche-prcheck-1872.apps.che-dev.x6e0.p1.openshiftapps.com
[pre-release-tests]

**Pull request merged**